packages/app/src/GoogleBtn.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import { GoogleLogin, GoogleLogout } from "react-google-login";
+
+const CLIENT_ID = "216565480006-kir74928iff14l4ln6p5kn0llrd5jk9a.apps.googleusercontent.com";
+
+class GoogleBtn extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLogined: false,
+      accessToken: ""
+    };
+
+    this.login = this.login.bind(this);
+    this.handleLoginFailure = this.handleLoginFailure.bind(this);
+    this.logout = this.logout.bind(this);
+    this.handleLogoutFailure = this.handleLogoutFailure.bind(this);
```

Comment:
I don't if this component changed, but you don't need have `bind` here if you change all of these functions to arrow `()=>` funcs.  

Also the state can be declared outside of the constructor, so the constructor can be deleted totally.